tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -250,6 +258,11 @@ def update_snapshot(self, data: bytes):
             raise RuntimeError("Cannot update snapshot after targets")
         logger.debug("Updating snapshot")
 
```

Comment:
```suggestion

        # client workflow 5.4.4: Make sure the final timestamp is not expired.
```
Maybe it will be even clearer if you add the step number of the client workflow?

Reply:
It would but unfortunately it also requires keeping these numbers updated with spec changes... so I'm more inclined to remove the step numbers I did use in the PR then I am to add more of them.